src/esp/geo/Geo.h

```diff
@@ -5,8 +5,12 @@
 #ifndef ESP_GEO_GEO_H_
 #define ESP_GEO_GEO_H_
 
+#include <set>
+#include <typeinfo>
```

Comment:
Maybe I missed it, but it this include necessary?

Reply:
yep.  using both sets for cc's and typeinfo for error message.